src/libprojectM/Makefile.am

```diff
@@ -12,7 +12,7 @@ AM_CPPFLAGS = \
 	$(FTGL_CFLAGS) $(CG_CFLAGS)
 
 # system headers/libraries/data to install
-include_HEADERS = projectM.hpp
+include_HEADERS = projectM.hpp Common.hpp dlldefs.h event.h fatal.h PCM.hpp
```

Comment:
👍

Reply:
So Common.hpp includes config.h as of 4/5.  Commit log says "Add FTGL configure flag".  I can remove config.h from Common.hpp but I would break whatever was being tried with the FTGL configure flag.   For places that need this flag, could we include config.h at the lower level files?

Reply:
I think that sounds like a fine idea, yes 👍

Reply:
https://github.com/projectM-visualizer/projectm/pull/48/files#r180543415